popSimManu.tex

```diff
@@ -293,7 +294,7 @@ \section*{Results}
 the \cite{salome2011recombination} map for \textit{A.~thaliana};
 the \cite{comeron2012many} map for \textit{D.~melanogaster},
 and the \cite{campbell2016pedigree} map for \textit{C.~familiaris}.
-Adding further maps to the library is straightforward.
+Adding further maps to the library is straightforward. %KEL: Should we allude to how to do this? 
```

Comment:
It's not documented - right now someone would do it by copying the existing structure for other maps (which should be straightforward) - see the response to reviewers. I think this is fine.